app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def calculate_team_total_rating(team: list):
+    total_rating = sum(player.get_rating() for player in team)
```

Comment:
you can rewrite code without **total_rating**

Reply:
Sounds logical)
Done!